params/config.go

```diff
@@ -464,6 +465,11 @@ func (c *ChainConfig) IsGingerbreadP2(num *big.Int) bool {
 	return isForked(c.GingerbreadP2Block, num)
 }
 
+// IsL2 returns whether num represents a block number greater than or equal to the L2 migration block (last block before L2)
+func (c *ChainConfig) IsL2Migration(num *big.Int) bool {
```

Comment:
would be important to specify for users where L2MigrationBlock is:

 * The last block of the L1, 
 or
 * the first block of the L2
 
Here the comment says "Last block of L1 => L2Block -1", but the check says otherwise.

Reply:
Updated the PR so that l2MigrationBlock refers to first block of L2 (this is different than before) as it seems cleaner. Updated comments and help strings to clarify this